packages/components/src/styles/basiq/theme.css

```diff
@@ -0,0 +1,194 @@
+/* Base styles */
+:root {
+  background: var(--background-color);
+  font-family: system-ui;
+  font-size: 14px;
+  line-height: 1.5;
+}
```

Comment:
These are very strong assumptions that I would probably set not in basiq but in a more advanced theme.

Reply:
But we need a default, right? Maybe make all use custom CSS Properties?

Reply:
Yeah we do need a default. I was thinking that if this is the basic css for the components only, maybe defaults for the website as a whole should not be here... 🤔

Reply:
yep, let's have one for the storybook, then only customize the very basic.
I will take care this evening.